src/pretix/control/templates/pretixcontrol/order/index.html

```diff
@@ -281,15 +281,21 @@ <h3 class="panel-title">
                                     <div class="position-buttons">
                                         {% if line.generate_ticket %}
                                             {% for b in download_buttons %}
-                                                <form action="{% url "control:event.order.download.ticket" code=order.code event=request.event.slug organizer=request.event.organizer.slug position=line.pk output=b.identifier %}"
-                                                        method="post" data-asynctask data-asynctask-download
-                                                        class="form-inline helper-display-inline">
-                                                    {% csrf_token %}
-                                                    <button type="submit"
-                                                            class="btn btn-xs btn-default">
+                                                {% if b.download_handled_by_frontend %}
+                                                    <a href="#" class="btn btn-default btn-xs btn-ticket-{{ b.identifier }}" target="_blank" data-organizer="{{ request.event.organizer.slug }}" data-event="{{ event.slug }}" data-order="{{ order.code }}" data-secret="{{ line.secret }}" data-position="{{ line.pk }}" data-generate-url="{% url "control:event.order.download.ticket" code=order.code event=request.event.slug organizer=request.event.organizer.slug position=line.pk output=b.identifier %}">
```

Comment:
``line.secret`` is used for ``data-secret`` here. However, the documentation states it is the ``line.web_secret`` in same cases and ``order.secret`` in others.